CMakeLists.txt

```diff
@@ -48,7 +48,7 @@ FIND_PACKAGE(LibLZMA REQUIRED)
 
 
 include_directories(SYSTEM ${LibLZMA_INCLUDE_DIRS})
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${LLVM_CXXFLAGS} ${LLVM_VERSION}")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${LLVM_CXXFLAGS} ${LLVM_VERSION} -fno-strict-aliasing")
```

Comment:
Did we determine if this is actually needed?  I thought the bug that we thought might be caused by it was actually a qemu bug?